### PR TITLE
feat(docs): add info on how to configure additional interfaces and DNS names

### PR DESCRIPTION
**Pull request by lloesche**
# Description

Info on how to configure additional interfaces and DNS names

# Code of Conduct

By submitting this pull request, I agree to follow the [code of conduct](https://resoto.com/code-of-conduct).

### CODE REVIEW

File: docs/getting-started/configuring-resoto.md [hunk 1]
```diff
@@ -366,6 +366,66 @@ Once one or more cloud providers have been configured the `collect_and_cleanup`
...
+
+Alternatively:
+
+```
+> config set resoto.core resotocore.api.web_hosts=["::", "0.0.0.0"]
+```
+
+## Add additional DNS names to the Resoto Core TLS certificate
+
+By default Resoto tries to find all local IP addresses and hostnames and adds them to the subject alternative names (SAN) list of the x509 certificate. If the system you are running Resoto on has additional DNS names you can add them to the SAN list using the `resotocore.api.host_certificate.san_dns_names` option of the `resoto.core` config.
```
Comment (TheCatLady) [inline docs/getting-started/configuring-resoto.md:400]:
```suggestion
Resoto attempts to find all local IP addresses and hostnames and add them to the subject alternative names (SAN) list of the x509 certificate. If your system has additional DNS names, you can add them to the SAN list via the `resotocore.api.host_certificate.san_dns_names` option in the `resoto.core` configuration.
```

File: docs/getting-started/configuring-resoto.md [hunk 2]
```diff
@@ -366,6 +366,66 @@ Once one or more cloud providers have been configured the `collect_and_cleanup`
...
+
+In `resh` enter:
```
Comment (TheCatLady) [inline docs/getting-started/configuring-resoto.md:402]:
```suggestion
Execute the following command in [Resoto Shell](../concepts/components/shell.md) to open the relevant configuration for editing:
```

File: docs/getting-started/configuring-resoto.md [hunk 3]
```diff
@@ -366,6 +366,66 @@ Once one or more cloud providers have been configured the `collect_and_cleanup`
...
+
+```
```
Comment (TheCatLady) [inline docs/getting-started/configuring-resoto.md:404]:
````suggestion
```bash
````

File: docs/getting-started/configuring-resoto.md [hunk 4]
```diff
@@ -366,6 +366,66 @@ Once one or more cloud providers have been configured the `collect_and_cleanup`
...
+      # Include loopback in certificate
+      include_loopback: true
+      # List of DNS names to include in CSR
+      san_dns_names:
+        - 'some.engineering'
+        - 'resoto.some.engineering'
+        - '*.some.engineering'
+      # List of IP addresses to include in CSR
+      san_ip_addresses: []
+[...]
```
Comment (TheCatLady) [inline docs/getting-started/configuring-resoto.md:426]:
```suggestion
  [...]
```

File: docs/getting-started/configuring-resoto.md [hunk 5]
```diff
@@ -366,6 +366,66 @@ Once one or more cloud providers have been configured the `collect_and_cleanup`
...
+
+By default Resoto tries to find all local IP addresses and hostnames and adds them to the subject alternative names (SAN) list of the x509 certificate. If the system you are running Resoto on has additional DNS names you can add them to the SAN list using the `resotocore.api.host_certificate.san_dns_names` option of the `resoto.core` config.
+
+In `resh` enter:
+
+```
+> config edit resoto.core
+```
+
+```yml title="Adding additional DNS names to the SAN list"
```
Comment (TheCatLady) [inline docs/getting-started/configuring-resoto.md:408]:
````suggestion
```yaml title="Resoto Core configuration with additional DNS names on SAN list"
````

File: docs/getting-started/installation/pip.md [hunk 6]
```diff
@@ -82,6 +82,8 @@ $ source ~/resoto/resoto-venv/bin/activate
 $ resotocore --graphdb-password "$graphdb_password" --graphdb-server http://localhost:8529 --psk "$pre_shared_key"
 ```
 
+By default `resotocore` listens on `localhost:8900`. If desired you can [make it listen on all interfaces](../configuring-resoto#make-resoto-core-listen-on-multiple-interfaces).
```
Comment (TheCatLady) [inline docs/getting-started/installation/pip.md:85]:
```suggestion
[Resoto Core](../../concepts/components/core.md) only listens on `localhost:8900` by default. [Resoto Core](../../concepts/components/core.md) can be [configured to listen on all interfaces](../configuring-resoto.md#configuring-resoto-core-network-interfaces) if desired.
```

**Review by TheCatLady — CHANGES_REQUESTED** [review]

File: docs/getting-started/configuring-resoto.md [hunk 7]
```diff
@@ -366,6 +366,66 @@ Once one or more cloud providers have been configured the `collect_and_cleanup`
 > workflow run collect_and_cleanup
 ```
 
+## Make Resoto Core Listen on Multiple Interfaces
+
+For security reasons `resotocore` by default only listens on the loopback interface. This can be changed using the `resotocore.api.web_hosts` option of the `resoto.core` config.
+
+In `resh` enter:
+
+```
```
[the remark this answers is not in the document]
Comment (lloesche) [reply docs/getting-started/configuring-resoto.md:375]: resh is not bash, let's not use bash syntax highlighting it creates confusing visualization!

**Review by TheCatLady — APPROVED** [review]

**Pull request merged**